src/events/scraper/run-scraper/index.js

```diff
@@ -1,3 +1,15 @@
+const assertTotalsAreReasonable = require('./scraper-helpers/assert-totals-are-reasonable')
+const buildGetIso2FromName = require('./scraper-helpers/build-get-iso2-from-name')
+const getKey = require('./scraper-helpers/get-key')
+const normalizeTable = require('./scraper-helpers/normalize-table')
+
+const scraperHelpers = {
+  assertTotalsAreReasonable,
+  buildGetIso2FromName,
+  getKey,
+  normalizeTable,
+}
```

Comment:
May just want to stick an index in helpers that exports all these but nbd either way